src/components/PercentBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { numericValue } from '../utils/displayUtils';
+
+export enum BarType {
+  Success = '#027d11',
```

Comment:
Ant design provides a color [palette](https://ant.design/docs/spec/colors). You could pull from that palette. I know in larger projects having a limited palette is nice, but here it doesn't really matter. Just wanted to present that option 😄

Reply:
I'm not generally a front-ender, so what I've done here is already an extension of my abilities. I've updated the colors used to match what's in the palette in 1608c24. If there's a better way to do the CSS styling on that, please feel free to modify it as needed.